sasHTTP/httpconnector.cpp

```diff
@@ -132,7 +132,7 @@ namespace SAS {
             }
 
 			ne_uri uri;
-            if(ne_uri_parse(options.baseURL.c_str(), &uri) != 0)
+            if(ne_uri_parse(options.baseURL.c_str(), &uri) != 0 || uri.scheme == NULL || uri.host == NULL)
```

Comment:
nem lenne célszerű megfordítani a logikát?
```suggestion
            if(uri.scheme == nullptr || uri.host == nullptr || ne_uri_parse(options.baseURL.c_str(), &uri) != 0)
```

Reply:
Az uri-t az ne_uri_parse tölti ki, így az eredeti sorrend a jó szerintem.

Reply:
..akkor így:
```suggestion
            if(ne_uri_parse(options.baseURL.c_str(), &uri) != 0 || !uri.scheme || !uri.host)
```
...ha olyan nem lehet, hogy a uri.scheme akaratlagosan nullptr